### ISSUE TITLE
NTR: Abdominal wall mass (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7875]
New term request Abdominal wall mass (elthat_100321031237):   UMLS: 1328363

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000171
- label: Abdominal wall mass
- parents: [Abnormality of the abdominal wall (HP:0004298)](https://hpo.jax.org/app/browse/term/HP:0004298)
- pmid: PMID:32330085
- def: An abnormal enlargement or swelling localized to the abdominal wall (the skin, fascia, and muscle encases the abdominal cavity and viscera). The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7875
- synonyms:
